src/index.ts

```diff
@@ -331,8 +332,8 @@ function initMappers(options: InternalResolverOptions) {
     typeof options.project === 'string'
       ? [options.project]
       : Array.isArray(options.project)
-      ? options.project
-      : [process.cwd()]
+        ? options.project
```

Comment:
Why are these lines changed?

Reply:
Not sure. The latest update should have reverted the change.